packages/store-engine/src/main/store/TransientStore.test.browser.ts

```diff
@@ -131,11 +131,10 @@ describe('store.TransientStore', () => {
 
     afterEach(() => jasmine.clock().uninstall());
 
-    it('publishes an event when an entity expires.', async done => {
-      store.on(TransientStore.TOPIC.EXPIRED, expiredBundle => {
+    it('publishes an event when an entity expires.', async () => {
+      await store.on(TransientStore.TOPIC.EXPIRED, expiredBundle => {
         expect(expiredBundle.payload).toBe(entity);
         expect(expiredBundle.primaryKey).toBe(primaryKey);
-        done();
       });
```

Comment:
I'm pretty sure the expects are not run while the test is running. 
Please fix this (you can probably remove the `async` from the function signature since it's not needed and keep the `done` callback)